src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDesignTimeDescriptorFactory.cs

```diff
@@ -38,13 +39,20 @@ public static TagHelperDesignTimeDescriptor CreateDescriptor([NotNull] Type type
                 ?.OfType<OutputElementHintAttribute>()
                 .FirstOrDefault();
             var outputElementHint = outputElementHintAttribute?.OutputElement;
+            var editorBrowsableAttribute = type
+                .GetCustomAttributes(inherit: false)
```

Comment:
Not looking down the inheritance chain here because `TargetElement` isn't inherited. However, the default inheritance behavior of `EditorBrowsableAttribute` is `true`. I'm open to opinions :smile: